library/src/main/java/com/bumptech/glide/util/ContentLengthInputStream.java

```diff
@@ -58,12 +62,12 @@ public synchronized int read() throws IOException {
   }
 
   @Override
-  public int read(@NonNull byte[] buffer) throws IOException {
```

Comment:
I prefer to not add nullability annotations to overridden methods who don't already have them, what do you think? It's making assumptions about third party codebases... (I know this is the java lang so it's safe, but as a general rule I prefer not to)

Reply:
Doesn't matter to me, I think it's ok either way.

Reply:
Sweet!